src/layout/List/ListLayoutValidator.tsx

```diff
@@ -0,0 +1,42 @@
+import { useEffect } from 'react';
+
+import { useLanguage } from 'src/features/language/useLanguage';
+import { NodesInternal } from 'src/utils/layout/NodesContext';
+import type { NodeValidationProps } from 'src/layout/layout';
+
+export function ListLayoutValidator(props: NodeValidationProps<'List'>) {
```

Comment:
Isn't this duplicated now? Might be better to make one that works the same for all components. If I'm not mistaken, it's as simple as:

```suggestion
export function LayoutValidatorForGroupAndChecked(props: NodeValidationProps<'List' | 'Checkboxes' | 'MultipleSelect'>) {
```